chia/consensus/blockchain.py

```diff
@@ -447,10 +498,11 @@ async def _reconsider_peak(
                 block_record.height,
                 records_to_add,
                 (list(lastest_coin_state.values()), hint_coin_state),
+                changes_for_peer,
```

Comment:
I'm pretty sure you no longer need to return the `lastest_coin_state` and `hint_coin_state`. The only reason (afaik) for this to be returned was for checking subscriptions later

Reply:
It is still being used by various other functions.